src/Reactor.hh

```diff
@@ -5,6 +5,7 @@
 #include "StringSetting.hh"
 #include "EventListener.hh"
 #include "Observer.hh"
+#include "MSXMotherBoard.hh"
```

Comment:
I am a bit concerned about this `#include`:
* `Reactor.hh` is included in many other files.
* `MSXMotherBoard.hh` is modified relatively frequently.

The combination of these two will cause many re-compilations on incremental builds (on any change in `MSXMotherBoarrd.hh`).

The reason for this `#include` is to make `enum SetupDepth` visible. I see two possibilities to improve the incremental compilation situation:
* Move this enum into a new header file. And #include that header instead.
* In `Reactor.hh` only specify a forward declaration of this enum. Somthing like `enum class SetupDepth : uint8_t;`. That should be sufficient for the uses of `SetupDepth` inside `Reactor.hh`.

Both require moving the enum out of the `MSXMotherBoard` class. That's not too bad IMHO.

Reply:
OK, changed it.